internal/commands/ip_access/update.go

```diff
@@ -0,0 +1,64 @@
+package ipaccess
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/spf13/pflag"
+)
+
+// CommandMetaUpdate is the command meta for the `ip-access update` command
+var CommandMetaUpdate = cli.CommandMeta{
+	Use:     "update",
+	Display: "allowed IP update",
```

Comment:
use `"ip-access update"` here

---

also need to add `HelpText` and `Description` here to be complete, work with @kpham1231 if need be

Reply:
will create a ticket and a `TODO`.